components/audio_board/Kconfig.projbuild

```diff
@@ -274,6 +287,24 @@ menu "Audio Board"
 	                This is labeled "SD" on chip/boards.
 	    endmenu
 
+		menu "PT8211 interface Configuration"
+	        depends on DAC_PT8211
+
+	        config PT8211_MUTE_PIN
+	            int "Master mute/unmute"
+	            default 18
```

Comment:
Is there a reason why you chose this as a default? Shouldn't it be -1?

Reply:
Ah good point. I just copied most of the code from another existing audio board. Also I think I should check for -1 before initializing and setting the GPIO...